tests/tlm/test_properties.py

```diff
@@ -3,15 +3,12 @@
 
 import pytest
 
-from cleanlab_studio.internal.constants import (
-    _VALID_TLM_MODELS,
-    _VALID_TLM_QUALITY_PRESETS,
-)
+from cleanlab_studio.internal.constants import _VALID_TLM_QUALITY_PRESETS
 from cleanlab_studio.studio.trustworthy_language_model import TLM
 from tests.tlm.test_get_trustworthiness_score import is_trustworthiness_score
 from tests.tlm.test_prompt import is_tlm_response
 
-valid_tlm_models = [model for model in _VALID_TLM_MODELS if "sonnet" not in model]
+valid_tlm_models = ["gpt-4o-mini", "gpt-3.5-turbo-16k", "gpt-4", "gpt-4o", "claude-3-haiku"]
```

Comment:
Sorry this is kinda nit, but I think we rather explicitly exclude that include right? Like if we add a new model, we want it to automatically be tested

Reply:
Oh I think I misunderstood your review here. I thought you wanted an explicit list of IN models. Yep